client/client.go

```diff
@@ -255,7 +255,15 @@ func (ec *SDKClient) Balance(
 		}
 		balance, err := decodeHexData(resp)
 		if err != nil {
-			return nil, err
+			// if the erc20 token dose exist we will get 0x 
+			if resp == "0x" {
+				log.Printf("eth_call return 0x at block %d, for account %s, with token symbol: %s and contract address: %s", *block.Index, account.Address, currency.Symbol, value.(string))
```

Comment:
We can remove this debugging message to avoid spam the log, only comments here are sufficient

Reply:
change it to fmt? I think it is better to record something as we do not return err here
BTW, the logs will send tons of /block and post /network/status, perhaps it is ok to log here
and this kind of situation is not common, so we will not see too many these kind of logs

Reply:
I check my local logs, and only 10 these logs in tons logs(> 0.5 million)

Reply:
For genetic logging system, we should only log the error and status info, for me this is more like debugging message and we should put it under logger debug mode, given currently our logger is not mature and we only support info mode, the best way is to remove this kind of log. Also even we hit this code path this log is still not useful as we return 0 directly here already

Reply:
Another thing I am thinking is that should be fix this in [decodeHexData](https://github.com/coinbase/rosetta-geth-sdk/blob/bed30e6be8c7b8ebdfaca51cfaca5e20ac72405c/client/client.go#L124) function instead of here? The reason is looks like `0x` is not an error and we just need to make [decodeHexData](https://github.com/coinbase/rosetta-geth-sdk/blob/bed30e6be8c7b8ebdfaca51cfaca5e20ac72405c/client/client.go#L124) accepts it and return `0`, rather than forcing decode it

Reply:
yes, the reason why I do not fix in that function is I want to print the log

Reply:
Basically the response `0x` from `eth_call` is correct, we just need to fix our decode function to interpret it properly instead of forcing to decode + return error IMO